resources/ctia-default.properties

```diff
@@ -44,7 +44,6 @@ ctia.store.incident=es
 ctia.store.indicator=es
 ctia.store.judgement=es
 ctia.store.relationship=es
-ctia.store.verdict=es
```

Comment:
Nevermind, we don't need this--the default properties ship with the package itself.